src/blue-magic/learnt-info.cpp

```diff
@@ -92,7 +92,7 @@ void learnt_info(PlayerType *player_ptr, char *p, MonsterAbilityType power)
     case MonsterAbilityType::BA_MANA:
     case MonsterAbilityType::BA_DARK:
     case MonsterAbilityType::BA_LITE:
-        set_bluemage_damage(player_ptr, power, plev, KWD_DAM, p);
+        comment = set_bluemage_damage(player_ptr, power, plev, KWD_DAM);
```

Comment:
This sentences can be changed:

Before:
```
comment = set_bluemage_damage(player_ptr, power, plev, KWD_DAM);
break;
```

After:
```
return set_bluemage_damage(player_ptr, power, plev, KWD_DAM);
```

Reply:
The forced push changed the case statement in learnt_info() so each case directly returns a result.